scripts/fund-address.ts

```diff
@@ -156,12 +154,18 @@ async function main() {
   const sender = chain.getPublicKey().toAddress().toString()
 
   // maybe fund ERC20
-  if (argv.erc20) {
-    await fundERC20(chain, sender, argv.address, argv.target)
-    return
+  if (argv.targetERC20) {
+    await fundERC20(chain, sender, argv.address, argv.targetERC20)
+  }
+
+  if (argv.targetNative) {
+    await fundNative(chain, sender, argv.address, argv.targetNative)
+  }
+
+  if (!argv.targetERC20 && !argv.targetNative) {
```

Comment:
rather check this in the beginning. I know it doesn't make a difference but it easier to comprehend.

Reply:
changed